src/js/owl.carousel.js

```diff
@@ -842,7 +843,7 @@
 				if (coordinate > value - pull && coordinate < value + pull) {
 					position = index;
 				} else if (this.op(coordinate, '<', value)
-					&& this.op(coordinate, '>', coordinates[index + 1] || value - width)) {
+					&& this.op(coordinate, '>', coordinates[index + 1] + pull || value - width)) {
```

Comment:
If you're only moving the `pull` attribute to options, why are we adding the`pull` value here now when we weren't before?